pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -129,22 +129,16 @@ func getConfigurationName(args []string) (string, error) {
 
 //  Get the configurator based on the ipfailover type.
 func getPlugin(name string, f *clientcmd.Factory, options *ipfailover.IPFailoverConfigCmdOptions) (ipfailover.IPFailoverConfiguratorPlugin, error) {
-	//  Currently, the only supported plugin is keepalived (default).
-	plugin, err := keepalived.NewIPFailoverConfiguratorPlugin(name, f, options)
-
-	switch options.Type {
-	case ipfailover.DefaultType:
-		//  Default.
-	// case <new-type>:  plugin, err = makeNewTypePlugin()
-	default:
-		return nil, fmt.Errorf("No plugins available to handle type %q", options.Type)
-	}
+	if options.Type == ipfailover.DefaultType {
```

Comment:
Minor style comment - we would generally invert this condition and exit early, so that the average indentation level of this function is lower.